src/test/run-make/big-vec/Makefile

```diff
@@ -0,0 +1,5 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) -C opt-level=0 big_vec.rs
```

Comment:
Could this be added as a `run-pass` test instead? We've got builders for unoptimized tests which should cover this I believe.